### PR TITLE
Sara / Updated signup new design

### PR DESCRIPTION
**Pull request by sara-fs**
# Description

Summary of the change and/or issue that is fixed.

Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/custom/_signup-new.js [hunk 1]
```diff
@@ -37,21 +38,23 @@ const Line = styled.div`
...
     width: 100%;
-    padding: 1.6rem 1.6rem;
+    height: 7.7rem;
+    padding: 0.8rem 0.8rem;
     margin-top: 1.6rem;
-    border-radius: 0.6rem;
-    background-color: var(--color-grey-8);
-    border: solid 1px var(--color-grey-17);
+    border-radius: 0.4rem;
+    background-color: rgba(242, 243, 244, 0.56);
```
Comment (arash-fs) [inline src/components/custom/_signup-new.js:57]:
```suggestion
    background-color: var(--color-grey-8);
    opacity: 0.56;
```

File: src/components/elements/checkbox.js [hunk 2]
```diff
@@ -35,7 +35,6 @@ const StyledCheckbox = styled.div`
         if (props.secondary && props.checked) return '2px solid var(--color-red)'
         return '2px solid var(--color-grey-5)'
     }};
-    border-radius: 3px;
```
Comment (arash-fs) [inline src/components/elements/checkbox.js:38]: we are using this element over the website, would you please change the `border-radius` inside the inheritance component if it's needed?

File: src/components/custom/_signup-new.js (hunk 1 above)
Comment (sara-fs) [reply src/components/custom/_signup-new.js:57]: I don't know why, but the result is not the same.

File: src/components/elements/checkbox.js (hunk 2 above)
Comment (sara-fs) [reply src/components/elements/checkbox.js:38]: `border-radius` has been removed from checkbox style throughout the project.

**Pull request closed without merging**